modules/ossm-config-control-plane-infrastructure-node-cli.adoc

```diff
@@ -3,12 +3,17 @@
 // * service_mesh/v2x/ossm-deployment-models.adoc
 
 :_content-type: PROCEDURE
-[id="ossm-config-control-plane-infrastructure-node_{context}"]
-= Configuring all {SMProductShortName} control plane components to run on infrastructure nodes
+[id="ossm-config-control-plane-infrastructure-node-cli_{context}"]
+= Configuring all control plane components to run on infrastructure nodes using the CLI
 
-This task should only be performed if all of the components deployed by the {SMProductShortName} control plane (including Istiod, Ingress Gateway, and Egress Gateway) along with optional elements (such as Prometheus, Grafana, and Distributed Tracing) are running on infrastructure nodes. 
+Perform this task if all of the components deployed by the {SMProductShortName} control plane, including Istiod, Ingress Gateway, and Egress Gateway, along with optional elements, such as Prometheus, Grafana, and Distributed Tracing, will run on infrastructure nodes. 
 
-If the control plane runs on a worker node, skip this task.
+If the control plane will run on a worker node, skip this task.
+
+.Prerequisites
+
+* You have installed the {SMProductName} Operator.
+* You must have an account with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
```

Comment:
```suggestion
* You are logged in as a user with the `cluster-admin` role. 
* If you use {product-dedicated}, you are logged in as a user with the `dedicated-admin` role.
```
Do these accounts just need to exist? Or does the user need to be logged in with one of these accounts? If the latter, edit per my suggestion.

Reply:
Done